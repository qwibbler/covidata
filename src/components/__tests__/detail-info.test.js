import React from 'react';
import renderer from 'react-test-renderer';
import DetailInfo from '../detail-info';

it('renders correctly', () => {
  const component = renderer.create(
    <DetailInfo infoName="Test: " infoNumber={100} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
