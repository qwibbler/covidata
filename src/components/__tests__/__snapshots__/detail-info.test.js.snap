// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="info bar"
>
  <div
    className="div-bg"
  />
  <p>
    Test: 
  </p>
  <p>
    100
  </p>
</div>
`;
